.github/workflows/tests.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install ".[dev]"
+          pip install '.[dev]'
```

Comment:
it keeps saying this doesn't have coverage installed, not that it the syntax is bad